Buildings/Fluid/Storage/Plant/StoragePlant.mo

```diff
@@ -0,0 +1,224 @@
+within Buildings.Fluid.Storage.Plant;
+model StoragePlant "Model of a storage plant with a chiller and a CHW tank"
+
+  replaceable package Medium =
+    Modelica.Media.Interfaces.PartialMedium "Medium package";
+
+  Buildings.Fluid.Sources.PropertySource_T chi(
```

Comment:
I'm concerned that this doesn't model the electrical power consumption of the local chiller, which would be a primary output of modeling a district system with a chilled water storage tank, unless a user copies the entire storage plant model and switches out this ideal source for a chiller model.  But this would be difficult to maintain if there are updates to the storage plant model.  A suggestion is to separate out the chiller model from the storage, network connection, and control models to make it easier for a user to model a chiller with power consumption for their specific application.  This would then be similar and consistent also with switching out the ideal source used in the example district model for the remote chiller.  I've proposed such a change in https://github.com/lbl-srg/modelica-buildings/tree/issue2859_dualSourceTES_differentChiller as a draft to demonstrate my thinking. There, the example model implements the ideal source as a chiller model, but the user can use the example as a template for how to use the storage plant model along with their specific chiller models for local and remote chillers.  What do you think?